app/views/about/index.html.erb

```diff
@@ -126,14 +126,14 @@
       <ul>
         <li><%= link_to _('API Resources'), apipie_apipie_path, :title => _('Automate Foreman via a simple and powerful API') %></li>
         <li><%= link_to _('Manual'), documentation_url, :rel => "external" %></li>
-        <li><%= link_to _('Wiki'), "http://projects.theforeman.org", :rel => "external" %></li>
+        <li><%= link_to _('Wiki'), external_link_path(type: 'wiki'), :rel => "external" %></li>
```

Comment:
This is an odd case. It linked to Redmine in general with the label wiki. Since there's an issue tracker below, this looks correct.